code/modules/vehicles/mecha/mecha_defense.dm

```diff
@@ -296,8 +296,6 @@
 
 /obj/vehicle/sealed/mecha/welder_act(mob/living/user, obj/item/W)
 	. = ..()
-	if(user in src.occupants) //Fuck with any attempts by the current user to self heal whilst piloting
```

Comment:
Superseded by adding TRAIT_HANDS_BLOCKED to occupants inside mechs so they cant use any items